masterlist.yaml

```diff
@@ -1111,7 +1101,6 @@ plugins:
       - 'PerkusMaximus_CombatPlus.esp'
     tag:
       - C.Regions
```

Comment:
Modifies a few region names which are unrelated to Update or USSEP so this was fine.